common/src/main/java/org/broadleafcommerce/common/resource/service/ResourceBundlingService.java

```diff
@@ -92,4 +92,12 @@
      */
     boolean checkForRegisteredBundleFile(String versionedBundleName);
 
+    /**
+     * returns bundle name that contain that file
+     * @param fileName file to find in the bungle
+     * @return first bundle name that has file, null otherwise
+     */
+    public List<String> findBundlesNameByResourceFileName(String fileName);
```

Comment:
```suggestion
    List<String> findBundlesNameByResourceFileName(String fileName);
```

`public` would be unnecessary here, since it's an interface